src/content/en/updates/posts/2016/06/performance-observer.md

```diff
@@ -0,0 +1,131 @@
+* third party performance analysis and visualization tools
+* performance assessment integrated into IDEs and other developer tools
+
+Access to this kind of timing data is already available in most major
+browsers for [navigation timing][navigation], [resource timing][resource],
+and [user timing][user]. 
+The newest addition is the [performance observer][perf observer]
+interface, which is essentially a streaming interface to gather low level
+timing information asynchronously, as it's gathered by the browser.
+This new interface provides a number of critical advantages over previous
+methods to access the timeline:
+
+* Today, apps have to periodically poll and diff the stored measurements,
+which is costly. This interface offers them a callback (i.e. no need to
+poll). As a result app using this API can be more responsive and more
+efficient.
+* It’s not subject to buffer limits (most buffers are set to 150 items
+by default), and avoids race conditions between different consumers that
+may want to modify the buffer.
+* Performance observer notifications are delivered asynchronously and
+the browser can dispatch them during idle time, to avoid competing with
+critical rendering work.
+
+*Beginning in Chrome release 52, the performance observer interface is
+enabled by default*. Let’s take a look at how to use it…
+
+{% highlight HTML %}
+<html>
+<head>
+  <script>
+    var observer = new PerformanceObserver(list => {
+      list.getEntries().forEach(entry => {
+        // Display each reported measurement on console
+        if (console) {
+          console.log("Name: "       + entry.name      +
+                      ", Type: "     + entry.entryType +
+                      ", Start: "    + entry.startTime +
+                      ", Duration: " + entry.duration  + "\n");
+        }
+      })
+    });
+    observer.observe({entryTypes: ['resource', 'mark', 'measure']});
+    performance.mark('registered-observer');
+
+    function clicked(elem) {
+      performance.measure('button clicked');
+    }
+  </script>
+</head>
+<body>
+  <button onclick="clicked(this)">Measure</button>
+</body>
+</html>
+{% endhighlight %}
+
+This simple page starts with a script tag defining some JavaScript code:
+
+* We instantiate a new `PerformanceObserver` object and pass an event
+handler function to the object constructor. The constructor initializes
+the object such that our handler will be called every time a new set
+of measurement data is ready to be processed (with the measurement
+data passed as a list of objects). The handler is defined here as an
+anonymous function that simply displays the formatted measurement data
+on the console. In a more real world scenario, this data might be stored
+in the cloud for subsequent analysis, or piped into an interactive
+visualization tool.
+* We register for the [types of timing events][types] we’re interested
+in (via the `observe` method) and call the [mark][mark] method
```

Comment:
Several items:
* Remove the parens around 'via the observe method'.
* End method names with (). Please do this throughout.
* The mark() method needs backticks.